### ISSUE TITLE
Standardize name conventions for qibo-client

### ISSUE THREAD

**Issue by scarrazza** [qiboteam/qibo-cloud-backends#42]
There are some discrepancies between the qibo-client naming and this package, e.g. device vs platform.

**Comment by BrunoLiegiBastonLiegi** [comment]
Yeah, also the `AWS` client (not in main yet) uses the `device` keyword, we might change `platform` to `device` and make it a standard.
